### ISSUE TITLE
deprecated supported features values which will be removed in HA Core 2025.1

### ISSUE THREAD

**Issue by blavak68** [Ekman/home-assistant-pure-i9#92]
I don't know if you noticed, but in addition to other warnings, I have this warning in the log:

```
Logger: homeassistant.helpers.entity
Zdroj: helpers/entity.py:1639
Prvýkrát sa vyskytol: 06:47:50 (2 výskytov)
Naposledy prihlásený: 06:47:50

Entity None (<class 'custom_components.purei9.vacuum.PureI9'>) is using deprecated supported features values which will be removed in HA Core 2025.1. Instead it should use <VacuumEntityFeature.PAUSE|STOP|RETURN_HOME|FAN_SPEED|BATTERY|SEND_COMMAND|STATE|START: 12668>, please create a bug report at https://github.com/Ekman/home-assistant-pure-i9/issues and reference https://developers.home-assistant.io/blog/2023/12/28/support-feature-magic-numbers-deprecation
```

**Comment by Ekman** [comment]
Thanks for reporting this, I appreciate it.

It has already been fixed in the latest version `2.0.1-rc.2`. Let me know if it persists and I'll take another look at it. https://github.com/Ekman/home-assistant-pure-i9/issues/71

**Comment by blavak68** [comment]
thx, now its OK
